receiver/awscontainerinsightreceiver/internal/gpu/dcgmscraper_test.go

```diff
@@ -207,7 +161,7 @@ func TestNewDcgmScraperEndToEnd(t *testing.T) {
 	mp, cfg, err := mocks.SetupMockPrometheus(targets...)
 	assert.NoError(t, err)
 
-	scrapeConfig := getScraperConfig(scraper.hostInfoProvider)
+	scrapeConfig := GetScraperConfig(scraper.HostInfoProvider)
```

Comment:
why `GetScraperConfig` gets called again here? It looks like scraper already has scraper config set in line144, and the same scraper config should be used to assign values for the testing

Reply:
Yeah, I can access the scraperConfig from the scraper itself, I will update it